packages/web/hooks/use-amplitude-analytics.ts

```diff
@@ -1,10 +1,11 @@
-import { useCallback, useEffect } from "react";
 import {
-  init as amplitudeInit,
-  identify,
   Identify,
+  identify,
```

Comment:
just alphabetizing

Reply:
Yes we have a lint that should consistently sort the order of the import statements and the order of the terms within each.